raphtory/src/search/mod.rs

```diff
@@ -117,9 +117,10 @@ impl<'graph, G: GraphViewOps<'graph>> IndexedGraph<G> {
         // reverse to sort by it
         schema.add_u64_field(fields::VERTEX_ID_REV, FAST | STORED);
         // add name
-        schema.add_text_field(fields::NAME, TEXT);
```

Comment:
while this does indeed fix the issue with searching for long strings, it breaks normal full-text search completely